journal-letters/modernized/15-considerations-on-time.adoc

```diff
@@ -1,14 +1,23 @@
 [short="Considerations on the Value of Time"]
 == A Friendly Address to My Neighbors of the Town of Colchester, and Others Whom It May Concern. Occasioned by Considerations of the Value of Time, and the Advantage of a Right Improvement of It.
 
-"`For you know perfectly, that the day of the Lord so comes as a theif in the night.`"
-
-"`Therefore let us watch, and be sober.`"
-
-"`Let us who are of the day, be sober, putting on the breast-plate of Faith and Love,
-and for a helmet, the Hope of Salvation.`"
-1 Thess. 5.
-
+[quote.scripture, , ]
+____
+For you know perfectly, that the day of the Lord so comes as a theif in the night.
+____
+
+[quote.scripture, , ]
+____
+Therefore let us watch, and be sober.
+____
+
+[quote.scripture, , 1 Thess. 5.]
```

Comment:
yeah, i played around a bit with this on my computer, and i think it's both more correct and looks slightly better to make all 3 of these `quote.section-epigraph`